src/EditPopup.js

```diff
@@ -0,0 +1,32 @@
+import React, {useCallback, useState} from 'react';
+import './Popup.css';
+
+
+const EditPopup = ({show, closeEdit, onSubmit}) => {
```

Comment:
This looks awfully close to the `Popup`. Take a look at the two and think about what's different; are they things that could be dynamic and passed in as props? If you try to maintain two versions, you have to remember to change both when changing to one. The rule of thumb is: if you ever find yourself reaching for copy-paste, ask yourself if an abstraction would be better.

Reply:
Understood, starting into it, I was still trying to understand how to get the information passed around. 
I will work on turning this into something usable in multiple cases.
When making a change like this, do I resubmit a pull request?

Reply:
I made the changes so it works with one file "popup" and reacts based on a boolean value. I initially went to use strings in case another popup came along... but figured I should ask about that implementation first. Would you just setup a map of form types that key to different things? a map of maps?

Reply:
So the question is what makes a popup a "popup" (FYI, these are more commonly referred to as "modals", but popup is fine)? If I were to answer that question, it'd be the ability to hide and show it, the darkening of the background, and the frame. The contents don't actually matter. If you wanted, you could make it take children that defined the contents. Then you'd have a reusable component that's very good at doing what it does.